fe/fe-core/src/main/java/org/apache/doris/clone/TabletSchedCtx.java

```diff
@@ -1102,4 +1088,14 @@ public String toString() {
         }
         return sb.toString();
     }
+
+    // Comparator to sort the replica with version count, asc
+    public static class VersionCountComparator implements Comparator<Replica> {
+        @Override
+        public int compare(Replica r1, Replica r2) {
+            long verCount1 = r1.getVersionCount() == -1 ? Integer.MAX_VALUE : r1.getVersionCount();
+            long verCount2 = r2.getVersionCount() == -1 ? Integer.MAX_VALUE : r2.getVersionCount();
```

Comment:
Integer.MAX_VALUE -> Long.MAX_VALUE

Reply:
This is intentional, because `compare()` return int. If I use Long.MAX_VALUE, it may be overflow.

Reply:
the type of verCount is long, so I think we just need to return 1, -1, 0 to support bigger scope of version count value

Reply:
Done